docs/release_notes.md

```diff
@@ -1,4 +1,7 @@
 # Subscriptions API Release Notes
+<div class='post-date'>26 August 2020 - amount</div> 
+Added limit to payment and oneoffpayment amount for FI merchants to 2000.
```

Comment:
Maybe "oneoffpayment" could be written as "one off payment" ?

Reply:
better yet "one-off payment"

Reply:
also have we really "added" limit or "updated" it?

Reply:
the limit was already there, so we actually didn't do anything